packages/node-app/src/utils/auth.utils.ts

```diff
@@ -1,66 +1,28 @@
-import axios from 'axios';
-
 import { Unit, User } from '../models/user.model';
 import TAClient from './ta.utils';
 
-const googleAuthInstance = axios.create({
-  baseURL: process.env.GOOGLE_OAUTH_URI,
-});
-
-const getUserInfo = async (tokenType, accessToken) => {
+export const getUserUnits = async (userEmail) => {
   try {
-    const res = await googleAuthInstance.get('/userinfo', {
-      headers: {
-        Authorization: `${tokenType} ${accessToken}`,
-        Accept: 'application/json',
-      },
-    });
-
-    if (res.status === 200) {
-      const userInfo = res.data;
-      const taSession = await TAClient.signIn();
-      const dbUserRecord = await User.findOne({ email: userInfo.email });
-      let taUser;
-
-      if (dbUserRecord?.taId) {
-        taUser = await TAClient.getPersonById(taSession, false, dbUserRecord.taId);;
-      } else {
-        const runtimePeople = await TAClient.getPeople(taSession, false);
-  
-        taUser = runtimePeople.find(
-          (user) => user.email === userInfo.email
-        );
-      }
-
-      if (!taUser) {
-        throw Error('user is not active');
-      }
+    let taUser;
+    const user = await User.findOne({ email: userEmail });
+    const taSession = await TAClient.signIn();
+
+    if (user) taUser = await TAClient.getPersonById(taSession, false, user.taId);
+    else {
+      const taPeople = await TAClient.getPeople(taSession, false);
+      taUser = taPeople.find((person) => person.email === userEmail);
+    }
```

Comment:
I think we are missing saving the `taId` on the user instance. Also, maybe we should not assume that the `taId` is always set if the user instance exist